backtrace-library/src/androidTest/java/backtraceio/library/BacktraceClientUniqueEventTest.java

```diff
@@ -400,48 +400,4 @@ public void uniqueEventUpdateAttributes() {
 
         assertEquals(expectedValue, backtraceClient.metrics.getUniqueEvents().getLast().getAttributes().get(expectedKey));
     }
-
-    @Test
-    public void uniqueEventEmptyAttributeValueShouldNotOverridePreviousValueOnUpdate() {
```

Comment:
those tests don't make sense. If the user changes the attribute, the new attribute value should be send to metrics. It can generate false positive data if we won't change the value